source/ptreferential/ptreferential.h

```diff
@@ -56,22 +56,24 @@ www.navitia.io
 using navitia::type::Type_e;
 namespace navitia{ namespace ptref{
 
-// Un filter est du type stop_area.uri = "kikoolol"
+// Filter is something like stop_area.uri = "kikoolol" or vehicle_journey.has_headsign("john")
```

Comment:
rephase as we don't understand that the "or" is not in the example but between the 2 examples. And bob is stronger than john.